server/controllers/index.ts

```diff
@@ -47,3 +52,23 @@ export abstract class BaseController extends FormWizard.Controller {
     return error
   }
 }
+
+Object.assign(FormWizard.Controller.validators, {
+  ukDate(value: FormWizard.Value): boolean {
+    if (value === '') return true
+    try {
+      return Boolean(parseDateInput(value))
+    } catch (e) {
+      return e === undefined
```

Comment:
I'm not quite sure how the error could be undefined 🤔 - The "blank input" is already covered by the if above, is this for something else? Is this any different than `return false` when there is an exception?

Reply:
this is gross isn't it? js requires the catch block to have a param. eslint requires it is used. i don't want to log something as boring as an incorrectly-typed date. since `e` is always going to be an error instance , this was the tidiest way i could think of to use the variable.

Reply:
Yeah, it is a bit gross...eslint in our way :|

I am not saying this is less gross but perhaps the following is a bit less misleading and clear that we return always `false` on error?:

```
      // eslint-disable-next-line @typescript-eslint/no-unused-vars
    } catch (_e) {
      return false
    }
```